soh/soh/Enhancements/debugger/colViewer.cpp

```diff
@@ -15,40 +16,27 @@ extern "C" {
 extern GlobalContext* gGlobalCtx;
 }
 
-enum class ColRenderSetting {
-    Disabled,
-    Solid,
-    Transparent,
-    NumSettings
-};
+enum class ColRenderSetting { Disabled, Solid, Transparent };
 
-std::string ColRenderSettingNames[] = {
+std::vector<std::string> ColRenderSettingNames = {
```

Comment:
This can stay an array if you pass the count to the function drawing the text. You can also use `const char*`

Reply:
I mean i could add a count variable. But i think like this it's much cleaner as you would not need to pass another variable. Performance wise it should not make any difference.